webodf/tools/odfRng2Config.js

```diff
@@ -22,28 +22,73 @@
  * @source: https://github.com/kogmbh/WebODF/
  */
 
-function aggregate(callback) {
-    return function (collection, individual) {
-        return collection.concat(callback(individual));
-    };
-}
+"use strict";
 
-function toArray(nodeList) {
-    "use strict";
-    return Array.prototype.slice.call(nodeList);
+var rngns = "http://relaxng.org/ns/structure/1.0";
+
+/**
+ * Check if a node is an element in the rng namespace and the given localName.
+ * @param {!string} localName
+ * @param {!Node} node
+ * @return {!boolean}
+ */
+function isRng(localName, node) {
+    if (node.nodeType !== 1) {
```

Comment:
Magic number should probably have a variable name (or, you could use the defined value on the Node tree, e.g., ``Node.ELEMENT_NODE``)